### PR TITLE
update dapp to use new sdk signatures

### PR DESCRIPTION
**Pull request by ironFe93**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -100,11 +100,7 @@ function App() {
             };
 
             // Get an RIF Relay RelayProvider instance and assign it to Web3 to use RIF Relay transparently
-            const relayingServices = new DefaultRelayingServices({
-                web3Instance: web3,
-                rskHost: '',
-                envelopingConfig: config
-            });
+            const relayingServices = new DefaultRelayingServices(web3);
```
Comment (antomor) [inline src/App.tsx:103]: 👍🏼

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -32,19 +33,15 @@ function Footer(props: FooterProps) {
...
                 ...smartWallet,
                 balance: `${Utils.fromWei(balance)} tRIF`,
                 rbtcBalance: `${Utils.fromWei(rbtcBalance)} RBTC`,
-                deployed:
-                    (await provider?.isSmartWalletDeployed(
-                        smartWallet.address
-                    )) || false
             };
-            return swWithBalance;
+            return swWithBalance as SmartWalletWithBalance;
```
Comment (antomor) [inline src/components/Footer.tsx]: why don't we define the type when we declare the variable at line 39?

**Review by antomor — CHANGES_REQUESTED** [review]
I haven't tested it yet, but it looks good to me for now. I've just left a couple of doubts, but I'd prefer to test it before approving.

File: src/components/Footer.tsx (hunk 2 above)
Comment (ironFe93) [reply src/components/Footer.tsx]: I had some issues with this. Basically in order to make this work, I would have to add the properties "balance" and "rbtcBalance" to the sdk interface for the smart wallet.

**Comment by antomor** [comment]
@ironFe93 it looks like CI is failing because of `prettier`. Could you please have a look at it?

**Pull request merged**